Documentation/cmdref/cilium-agent.md

```diff
@@ -45,6 +45,7 @@ cilium-agent [flags]
       --bpf-policy-map-max int                               Maximum number of entries in endpoint policy map (per endpoint) (default 16384)
       --bpf-root string                                      Path to BPF filesystem
       --bpf-sock-rev-map-max int                             Maximum number of entries for the SockRevNAT BPF map (default 262144)
+      --bpf-socket-lb-hostns-only                            Skip socket LB for services when inside a pod namespace, in favor of service LB at the pod interface. Socket LB is still used when in the host namespace.
```

Comment:
Most other LB options start with: `--bpf-lb-...`. Given it's user-facing, could we name this: `--bpf-lb-hostns-only` ? This term is more generic but still fits given N-S and E-W LB happen both in hostns here.

Reply:
`--bpf-lb-hostns-only` is a bit misleading? since we still have bpf_lxc which does LB when this option is on?

Reply:
It's still attached inside hostns, but your'e right; definitely open to suggestions.. maybe `--bpf-lb-sock-hostns-only` would be somewhat better.

Reply:
Renamed to `--bpf-lb-sock-hostns-only`.